<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SourceFormatterClass</class>
 <widget class="QMainWindow" name="SourceFormatterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OneNote Format Helper</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="CLIPBOARD_BTN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Watch Clipboard ON</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>SourceCode</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>331</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="CPP_CB">
        <property name="text">
         <string>CPP</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="JAVA_CB">
        <property name="text">
         <string>JAVA</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="CSHARP_CB">
        <property name="text">
         <string>C#</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="XML_CB">
        <property name="text">
         <string>XML</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>HeadLines</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="H1_RB">
        <property name="text">
         <string>H1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="H2_RB">
        <property name="text">
         <string>H2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="H3_RB">
        <property name="text">
         <string>H3</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="HT_RB">
        <property name="text">
         <string>HT</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="OFF_RB">
        <property name="text">
         <string>OFF</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>433</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Source Code Highlighter</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SourceFormatter.qrc"/>
 </resources>
 <connections/>
</ui>
